README.md

```diff
@@ -16,12 +16,73 @@ A flexible JSON object linter with out of the box support for OpenAPI Specificat
 
 ## Installation
 
-```shell
-npm install @stoplight/spectral
+```bash
+npm install -g @stoplight/spectral
+```
+
+or
+
+```bash
+yarn global add @stoplight/spectral
 ```
 
 Supports Node v8.3+ and modern browsers.
 
+If you don't want to install Spectral, but still would like to try it out, you can do so as follows:
+
+```bash
+npx @stoplight/spectral lint path-to-your-file.yml
```

Comment:
Didn't test it out, but should work. To be verified once the package is published.

Reply:
At this point are we not just teaching people how npm/npx work? Do we need npm AND yarn AND npx? Its getting rather large.

Reply:
Yeah, I agree. I also feel it's way too broad.

Reply:
@philsturgeon re-arranged this a bit.